src/components/OurOffer/ComparisonTable/Table/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Typography } from "@mui/material";
+import { IFeature, IPlan, ITextProps } from "../../helpers";
+import classnames from "classnames";
+import CheckIcon from "@assets/icon/check-in-icon.svg?react";
+import CrossIcon from "@assets/icon/x-icon.svg?react";
+import useStyles from "./styles";
+
+interface IComparisonTableProps {
+  text: ITextProps;
+  features: IFeature[];
+  plans: IPlan[];
+}
+
+function Table({ features, plans }: IComparisonTableProps) {
+  const { classes } = useStyles();
+
+  function handleFeatureName(feature: IFeature) {
+    return (
+      <Typography
+        className={classnames(classes.tableFeature, {
+          [classes.emptyFeatureCell]: feature.starter === "",
+          [classes.nonEmptyFeatureCell]: feature.starter !== "",
+        })}
+      >
+        {feature.name}
+      </Typography>
+    );
+  }
+
+  function handlePlanTable(feature: IFeature, planTitle: string) {
```

Comment:
In my opinion, this has enough complexity to be refactored into a separate component. Additionally, I'm not a big fan of nested ternaries. I would rather go with this:

```
if (typeof value === 'string' && value !== '') {
  return <Typography>{value}</Typography>
} else if (value) {
  return <CheckIcon />
} else if (value === '') {
  return null
} else {
  return <CrossIcon />
}
```

Looks a lot more readable to me. 🙂